Project.toml

```diff
@@ -17,7 +18,7 @@ StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 
 [compat]
 FillArrays = "0.8"
-PDMats = "0.9.12"
+PDMats = "0.10"
```

Comment:
why did this one need to change? can we still support `0.9`?

Reply:
`0.9` causes the equality tests for the `MvNormal` Distributions to fail.